visidata/loaders/_pandas.py

```diff
@@ -17,6 +17,57 @@ def open_dta(vd, p):
     if not getattr(VisiData, funcname, None):
         setattr(VisiData, funcname, lambda vd,p,ft=ft: PandasSheet(p.name, source=p, filetype=ft))
 
+@VisiData.api
+def save_dta(vd, p, *sheets):
+    import pandas as pd
+    import numpy as np
+
+    fh = open("save_dta.log", "w", encoding="utf8")
+
+    #for vs in sheets:
+    vs = sheets[0]
+
+    columns = [col.name for col in vs.visibleCols]
+    fh.write(f"columns={columns}\n")
+    fh.flush()
+
+    types = list()
+    for r_i, dispvals in enumerate(vs.iterdispvals(format=True)):
```

Comment:
This loop unnecessarily goes over all items when all you want is the first item.  You could use a `break` instead of a `continue`, but really it should just be `dispvals = next(vs.iterdispvals())`.  If it were a passed-in iterator instead of `iterdispvals()` (which is restartable), then we'd want to reuse the first value instead of starting the loop over again below.

Reply:
I would use `vs.nRows`.